dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/org/apache/cxf/jaxws/ServiceImpl.java

```diff
@@ -208,6 +234,13 @@ private void initializePorts() {
                     } else if (e instanceof HTTPAddress) {
                         address = ((HTTPAddress)e).getLocationURI();
                     }
+                } else {
```

Comment:
May not need the else statement with trace (lines 237 to 241) as we are logging address anyway on line 243.